lib/content-services/src/lib/content-user-info/content-user-info.component.html

```diff
@@ -38,72 +38,64 @@
 
     <mat-menu #menu="matMenu" id="user-profile-lists" [xPosition]="menuPositionX" [yPosition]="menuPositionY"
               [overlapTrigger]="false" class="adf-userinfo-menu">
-        <mat-tab-group id="tab-group-env" (click)="stopClosing($event)" selectedIndex="0" role="menuitem"
```

Comment:
Also here, isn't this a rework of the user-info for content?

Reply:
same situation